app/views.py

```diff
@@ -65,21 +120,192 @@ def choose_transcription(request: HttpRequest) -> HttpResponse:
+            context.update(
+                {"ireddit_content_url": submission.content_url.split("/")[-1]}
+            )
+        elif "imgur.com" in submission.content_url:
+            context.update({"imgur_content_url": submission.content_url.split("/")[-1]})
+
+        # if they posted a bad transcription and we want them to fix it, grab
+        # the stored values and add them to the current context, then delete them
+        # from the session so they don't get accidentally shown twice.
+        if s_id := request.session.get("submission_id"):
+            if s_id == submission_id:
+                # it's for the submission we're currently working on.
+                if transcription := request.session.get("transcription"):
+                    context.update({"transcription": transcription})
+
+                if heading := request.session.get("heading"):
+                    context.update({"heading": heading})
+
+                if issues := request.session.get("issues"):
+                    context.update(
+                        {
+                            "issues": [
+                                f"app/partials/errors/{issue}.partial"
+                                for issue in issues
+                            ]
+                        }
+                    )
+            else:
+                # it's old data. Nuke it.
+                del request.session["transcription"]
+                del request.session["heading"]
+                del request.session["issues"]
+                del request.session["submission_id"]
+
+        return render(request, "app/transcribing.html", context)
+
     def post(self, request: HttpRequest, submission_id: int) -> HttpResponse:
         """Handle a submitted transcription."""
-        ...
+        transcription: str = request.POST.get("transcription")
+        transcription = transcription.replace("\r\n", "\n")
+
+        issues = check_for_formatting_issues(transcription)
+        if len(issues) > 0:
+            # stash the important stuff in the session so that we can
+            # retrieve it on the other side
+
+            # escape the backticks so that they can be rendered properly by the JS
+            # template literals in the html template
+            request.session["transcription"] = transcription.replace(
+                "`", "\`"  # noqa: W605
+            )
+            request.session["heading"] = request.POST.get("transcription_type")
+            request.session["issues"] = list(issues)
+            # store this one too so that we know which submission this information is for
+            request.session["submission_id"] = submission_id
+            return redirect("transcribe_submission", submission_id=submission_id)
+        else:
+            # we're good to go -- let's submit it!
+            if check_for_fenced_code_block(transcription):
+                transcription = clean_fenced_code_block(transcription)
```

Comment:
I removed it from the list specifically because we can correct for it automatically. https://github.com/GrafeasGroup/blossom/pull/188/files#diff-215ed3837d51fbabed7f0b470877d7dce92a65dff35964d495482c32d89874f8R166-R169